Documentation/guides/cpp_cmake.rst

```diff
@@ -29,18 +29,25 @@ Preparation
 
 #. Base NuttX compilation changes
 
-    For this example, load the configuration 'stm32f4discovery:testlibcxx' for building
+    For this example, load the configuration 'stm32f4discovery:nsh' for building
 
     .. code-block:: console
 
        $ cd nuttx
-       $ ./tools/configure.sh stm32f4discovery:testlibcxx
+       $ ./tools/configure.sh stm32f4discovery:nsh
 
     In menuconfig, the main points to be changed on a typical NuttX configuration are the following:
 
-    * Set RTOS Features -> Tasks and Scheduling -> Application entry point to 'hellocpp_main'
+    * RTOS Features -> Tasks and Scheduling -> Application entry point to 'main'
```

Comment:
Need to add `* Board Selection -> Board common logic -> to enable board common start up logic` otherwise you should have problems compiling non?

Reply:
Not really: the user can choose to init its board manually. I tried on xmc4800 without it and the output is OK.

Reply:
> Not really: the user can choose to init its board manually. I tried on xmc4800 without it and the output is OK.

can you try with the st please. It should pop an error.

Reply:
Working also with stm32f4discovery:nsh (although I cannot test it on hardware since I do not have it)

Reply:
I'll test again at home and get back to you :)